day_7/fizzbuzz.rb

```diff
@@ -0,0 +1,12 @@
+(1..100).each do |i|
```

Comment:
Nice Fizzbuzz! I recommend trying your hand at all of the Day-7 exercises this weekend to keep building that Ruby muscle. You can time box them to 2 hours each for an extra challenge and to prepare for your weekly challenges at Turing.